tests/e2e/lib/subscription_helpers.go

```diff
@@ -39,6 +30,27 @@ func (d *DpaCustomResource) GetOperatorSubscription(c client.Client, stream stri
 	return &Subscription{&sl.Items[0]}, nil
 }
 
+func (d *DpaCustomResource) GetOperatorSubscription(c client.Client, stream string) (*Subscription, error) {
+	err := d.SetClient(c)
+	if err != nil {
+		return nil, err
+	}
+
+	label := ""
+	if stream == "up" {
+		label = "operators.coreos.com/oadp-operator." + d.Namespace
+	}
+	if stream == "down" {
+		label = "operators.coreos.com/redhat-oadp-operator." + d.Namespace
+	}
+	return GetOperatorSubscription(c, d.Namespace, label)
+}
+
+func (v *VirtOperator) GetOperatorSubscription() (*Subscription, error) {
```

Comment:
I think this only works because virt_helpers and subscription_helpers are in the same package? I am pretty sure Go doesn't otherwise let you add arbitrary functions to types like this, so it might not be applicable everywhere.